README.md

```diff
@@ -89,6 +89,20 @@ const app = new App({
 })
 ```
 
+## Using with GitHub Enterprise
+
+The `endpointDefaults` option allows to override defaults from [octokit/endpoint](https://github.com/octokit/endpoint):
+
+```js
+const app = new App({
+  id: APP_ID,
+  privateKey: PRIVATE_KEY,
+  endpointDefaults: {
+    baseUrl: 'https://api.github-enterprise.com'
+  }
+}) 
```

Comment:
I wonder if we unnecessarily expose an implementation detail here 🤔 Would there be other options that one might want to pass? Maybe just adding a `baseUrl` option would be sufficient, as it will be the most common option people will need?